src/listmode_buildblock/CListModeDataROOT.cxx

```diff
@@ -317,6 +323,9 @@ check_scanner_definition(std::string& ret)
        if (max_num_non_arccorrected_bins == -1)
            stream << "Maximum number of non-arc-corrected bins := \n";
 
+       if (default_num_arccorrected_bins == -1)
+           stream << "Default number of arc-corrected bins := \n";
```

Comment:
Here, I could do 

```
if (max_num_non_arccorrected_bins == -1)
    default_num_arccorrected_bins = max_num_non_arccorrected_bins;
```
as a method of setting a default. It isnt easy to keep this optional.